### DISCUSSION TITLE
Live-reload in production for over-the-air updates?

### DISCUSSION THREAD

**Discussion by jqhoogland** [ionic-team/capacitor#5183] [Q&A] [answered]
What's stopping me from, for example, serving the javascript for an app at `mobile.domain.com` and publishing a capacitor app with the following in its `capacitor.config.js`:

```
server: {
    allowNavigation: ["domain.com"],
    hostname: "domain.com",
    url: "mobile.domain.com",
    cleartext: true,
  },
```

[The docs](https://capacitorjs.com/docs/v2/guides/live-reload#live-reload) are pretty explicit that you shouldn't do this:

> Be careful not to commit the server config to source control.

So I'm wondering why.

**Comment by jcesarmobile** [comment, marked as answer]
That doc is about live reload, that's (technically) a different thing, it's when you load your app from a local dev server, so you have to be careful to undo the changes because if another person clones the app and try to run without a local dev server running or if it has a different local IP it will fail to load.

What you want to do is to point to an existing website, you can do that with just the [url](https://capacitorjs.com/docs/config) property of the server object, but we also document it as "not intended for use in production", because it can lead to app rejections. It's not live reload because it doesn't reload automatically if you deploy a change on the web.

There are people using it, see https://github.com/ionic-team/capacitor/discussions/4080

**Reply by jqhoogland** [reply]
Thanks that's exactly what I was looking for!
